modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/feature/flag/FeatureFlagFactory.java

```diff
@@ -34,65 +32,19 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Properties;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.function.Function;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-
 /**
- * @author Drew Brokke
+ * @author Thiago Buarque
```

Comment:
You probably just want to add you name, not remove Drew's, right?

Reply:
Turns out is because git checks for the files similarity and then marks it as renamed. So, since I moved most of the content from `FeatureFlagsBagProvider` to `FeatureFlagsBagFactory`, git marks it as renamed.

See [StackOverflow answer](https://stackoverflow.com/questions/15031576/how-can-i-prevent-git-from-thinking-i-did-a-rename)

Reply:
Fixed